test/TRE/TestNoChange.cs

```diff
@@ -10,14 +11,14 @@ namespace UK.Gov.NationalArchives.CaseLaw.TRE.Test
 {
     public class TestNoChange
     {
+        private static readonly int[] FailingTestsToSkip = [6, 8, 31, 32];
+        public static readonly TheoryData<int> PassingTests = new(Tests.Indices.Except(FailingTestsToSkip));
 
-        public static readonly System.Collections.Generic.IEnumerable<object[]> Indices = Tests.indices;
-
-        // [Theory]
-        // [MemberData(nameof(Indices))]
-        public static void TestJudgments(int i)
+        [Theory]
+        [MemberData(nameof(PassingTests))]
```

Comment:
Is it fair to say that

Fact -> non-parametrized tests
Theory -> parametrized tests
?

Reply:
Exactly that 👍 
https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/